openmc/mgxs/mdgxs.py

```diff
@@ -581,6 +637,22 @@ def print_xs(self, subdomains='all', nuclides='all', xs_type='macro'):
             print(string)
             return
 
+        # Set polar/azimuthal bins
```

Comment:
Consider using a default `num_polar` and `num_azimuthal of unity and simplify this entire block as:

```python
polar_bins = np.linspace(
    0., np.pi, num=self.num_polar + 1, endpoint=True)
azimuthal_bins = np.linspace(
    -np.pi, np.pi, num=self.num_azimuthal + 1, endpoint=True)
```